compiler/rustc_mir/src/transform/check_unsafety.rs

```diff
@@ -580,24 +580,23 @@ fn is_enclosed(
     tcx: TyCtxt<'_>,
     used_unsafe: &FxHashSet<hir::HirId>,
     id: hir::HirId,
-) -> Option<(String, hir::HirId)> {
+    unsafe_fn_allowed: bool,
```

Comment:
This sounds like it controls whether unsafe functions are allowed. As unwieldy as it is, the name `unsafe_op_in_unsafe_fn_allowed` of the function computing this is much more clear.